defaults/main.yml

```diff
@@ -169,17 +169,23 @@ nifi_test_if_publicly_accessible: false
 
 nifi_backup_enabled: True
 
-nifi_backup_schedule: "0 0 * * 0"
+nifi_backup_id: "nifi.{% if 'nifi' in groups %}{{ groups['nifi'].index(inventory_hostname) + 1 }}{% else %}{{ groups['all'].index(inventory_hostname) + 1 }}{% endif %}"
+
+nifi_backup_schedule: "0 0 * * *" # Midnight
 nifi_backup_max_age: "7D"
+
+nifi_safe_backup_schedule: "0 21 * * 1" # 9PM Monday
```

Comment:
If you've got a better default for this just LMK or just change it - this may be nice as the americas are online but EMEA is finished.  When would you most like to get a "server down" notification when nifi fails to cleanly restart?  My choice was Monday night.